packages/hawtio/src/plugins/camel/routes-service.ts

```diff
@@ -205,6 +256,102 @@ class RoutesService {
   async deleteRoute(objName: string) {
     await jolokiaService.execute(objName, 'remove()')
   }
+
+  async dumbRoutesStatsXML(routesNode: MBeanNode | null): Promise<string | null> {
+    if (!routesNode) {
+      throw new Error('No route/routes node provided')
+    }
```

Comment:
if it doesn't accept `null` `routesNode`, you shouldn't make it `MBeanNode | null`. and you can remove the if guard.